### PR TITLE
Updated documentation, fixed flake-8, added README

### LINKED ISSUE #122: Update Basic Usage docs section
I've been following [Basic Usage](https://www.edgedb.com/docs/clients/00_python/usage#basic-usage) section of the EdgeDB documentation and this error occurred `edgedb.errors.InvalidReferenceError: type 'default::local_date' does not exist`.

I can see `local_date` was [moved](https://github.com/edgedb/edgedb/blob/master/docs/changelog/1_0_a2.rst#new-cal-module) to the `cal` module, so could you please update the [documentation](https://github.com/edgedb/edgedb-python/blob/master/docs/usage.rst#basic-usage) with `cal::local_date` like you did in [edgedb-python README](https://github.com/edgedb/edgedb-python/blob/master/README.rst#basic-usage)

If you don't mind I can do this on my own with PR.

---
By the way I can see the same thing in [edgedb README](https://github.com/edgedb/edgedb/blob/master/README.md#modern-type-safe-schema)

### PR DESCRIPTION
**Pull request by tech-cobber**
In this pull request:
- updated documentation with adding `cal::` before `local_date`, `local_time`, `local_datetime` in usage examples and types section, closes #122 
- added `README.md` in `docs/`. I think adding readme in documentation will help you attract more contributors, readme like this one helped me ones to start contributing. Feel free to decline this change.
- fixed flake-8 `B014 Redundant exception types`. After running `edb test` I had this:
image

    It might be intended to except all these errors, but the tests were failing and I had to do something with this, so I think it's better to fix rather than `# noqa: B014`. Feel free to decline this change.

### CODE REVIEW

File: docs/README.md [hunk 1]
```diff
@@ -0,0 +1,42 @@
+Documentation
+==================
```
Comment (1st1) [inline docs/README.md:2]: Everything looks good. One nit: please make underlines as long as the titles, not shorter and not longer.

File: docs/README.md [hunk 2]
```diff
@@ -0,0 +1,42 @@
...
+
+About
+------------
```
Comment (1st1) [inline docs/README.md:5]: ditto

File: docs/README.md (hunk 1 above)
Comment (tech-cobber) [reply docs/README.md:2]: Thank you, I didn't notice

File: docs/README.md (hunk 2 above)
Comment (tech-cobber) [reply docs/README.md:5]: +

**Comment by tech-cobber** [comment]
@1st1, thank you for the review

**Comment by 1st1** [comment]
@tech-cobber Oh, I actually pushed a fix for the linter errors and now the PR needs rebasing. Please do that, and also squash all commits into one. Thanks!

**Review by 1st1 — APPROVED** [review]

**Comment by tech-cobber** [comment]
I can see you approved, but I haven't done rebase and squash yet, please don't merge

**Comment by tech-cobber** [comment]
Oh...

**Comment by tech-cobber** [comment]
@1st1  I guess I will open another PR, sorry, I've done rebase wrong :C

File: edgedb/blocking_con.py [hunk 3]
```diff
@@ -29,7 +29,7 @@
 
 from .datatypes import datatypes
 from .protocol import blocking_proto
-from .protocol import protocol
+from .protocgitol import protocol
```
Comment (elprans) [inline edgedb/blocking_con.py:32]: Looks like an unintended change here.

**Comment by elprans** [comment]
>  I guess I will open another PR, sorry, I've done rebase wrong

The PR is fine, but there's a typo in one of the files.  Fix that and we'll squash-and-merge this.  Thanks!

File: edgedb/blocking_con.py (hunk 3 above)
Comment (tech-cobber) [reply edgedb/blocking_con.py:32]: I was talking about Yury's commits in my PR, if it's fine I'll just fix this change

**Comment by tech-cobber** [comment]
Thank you C:

**Pull request merged**